app/assets/javascripts/application.js

```diff
@@ -96,7 +96,7 @@ function order_by_occurrence(arr) {
           Quagga.initialized = true;
           Quagga.start();
       });
-  
+
```

Comment:
If this is an issue I am happy to remove, My editor auto removes trailing white space.

Reply:
np!